src/presence_analyzer/views.py

```diff
@@ -102,3 +102,27 @@ def presence_start_end_view(user_id):
     ]
 
     return days
+
+
+@app.route('/mean_time_weekday')
```

Comment:
Thats okay, but maybe we can think of something more generic, instead of creating 3 views? Don't ask me yet what do I mean, think it through.